test/integration.rb

```diff
@@ -5,13 +5,23 @@
 require 'test/unit'
 
 TEMP_PREFIX = 'svfs-test'
+TEMP_CONTAINER = "#{TEMP_PREFIX}.#{SecureRandom.uuid}"
+TEST_DIRECTORY = "#{ENV['TEST_MOUNTPOINT']}/#{TEMP_CONTAINER}"
+OLD_NAME = "#{TEST_DIRECTORY}/#{TEMP_PREFIX}"
```

Comment:
In fact `OLD_NAME` = `NEW_NAME` so it's not necessary to declare 2 global variables.
Also, we consider `old_name` and `new_name` to be the full path to an object, so consider naming this variable BASE_PATH instead.